src/data/mainnet/tokens-info.json

```diff
@@ -463,5 +463,12 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/USDC.png",
     "name": "Axelar Wrapped USDC",
     "symbol": "axlUSDC"
+  },
+  {
+    "address": "0x62b8b11039fcfe5ab0c56e502b1c372a3d2a9c7a",
+    "decimals": 18,
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/G$.png",
```

Comment:
Could the `$` in the URL potentially break something? AFAIK it's considered a special character (and is converted by some(?) browsers to its ASCII encoding `%24`).

Reply:
ugh, i was torn because I was trying to be consistent with the other asset names, but worried about how annoying `$` is to work with in file names.  I'll just rename it.